examples/search/analysis.ini

```diff
@@ -172,19 +172,22 @@ sngl-ranking = newsnr_sgveto_psdvar
 randomize-template-order =
 statistic-files = ${resolve:./statHL.hdf} ${resolve:./statLV.hdf} ${resolve:./statHV.hdf} ${resolve:./statHLV.hdf}
 
-[coinc-full_data]
+[coinc-defaultvalues]
 timeslide-interval = 0.1
 
+[coinc-full_data]
+
 [coinc-full_data-2det]
 loudest-keep-values = [-1:5,1:5]
 
 [coinc-full_data-3det]
 loudest-keep-values = [-3:5,-1:5]
+timeslide-interval = 0.11
```

Comment:
Is this just here as a demonstration, not as a scientific change? ... I have noticed some "demonstration settings" from here getting into places they shouldn't, so just need to be careful with this :-)

Reply:
This is just a demo